src/uniprotkb/adapters/proteinHighlights.ts

```diff
@@ -33,31 +33,31 @@ const highlightToEntrySection: {
   };
 } = {
   [highlightSection.domains]: {
-    link: `#${EntrySectionIDs[EntrySection.Function]}`,
+    link: `#${getEntrySectionNameAndId(EntrySection.Function).name}`,
```

Comment:
I guess related to the comment I left just a few seconds before. You might want to use id to use in a hash link

Reply:
Okay, makes sense. I used the name because that's what we used previously, but it won't work anymore!